### PR TITLE
Adding actions to the Project Updater

### PR DESCRIPTION
**Pull request by lancewf**
### :nut_and_bolt: Description

Adding the actions to the ingest-service's project updater. 

### :athletic_shoe: Demo Script / Repro Steps

1. `build components/ingest-service && start_all_services`
1. Add some actions `chef_load_actions 5`
1. Write down a few of the actions 'organization_name' values from elasticsearch with `curl $ELASTICSEARCH_URL/actions-*/_search | jq . | less`
1. Update the project ingest rules. Add the below code to here (https://github.com/chef/automate/blob/master/components/authz-service/server/v2/project.go#L248). Replace the `Values` with the 'organization_name' values found above. 
```
		ProjectRules: map[string]*api.ProjectRules{
			"project9": {
				Rules: []*api.ProjectRule{
					{
						Type: api.ProjectRuleTypes_EVENT,
						Conditions: []*api.Condition{
							{
								Type:   api.ProjectRuleConditionTypes_CHEF_ORGS,
								Values: []string{"The Great Lakes Avengers", "Howling Commandos", "X-Factor"},
							},
						},
					},
				},
			},
		},
```
1. Update the running authz component with `rebuild components/authz-service`
1. Test that the ingest rules are updated with `chef-automate dev grpcurl authz-service -- chef.automate.domain.authz.v2.Projects.ListProjectRules`
1. Start the project update process by creating a project and updating it. 
1. Create the project with `chef-automate dev grpcurl authz-service -- chef.automate.domain.authz.v2.Projects.CreateProject -d '{"name": "project9", "id": "project9"}'`
1. Update the project with `chef-automate dev grpcurl authz-service -- chef.automate.domain.authz.v2.Projects.UpdateProject -d '{"name": "project9", "id": "project9"}'`
1. Check that the project tags were updated on the actions in elasticsearch with `curl $ELASTICSEARCH_URL/actions-*/_search | jq . | less`

### :chains: Related Resources

completes - https://github.com/chef/automate/issues/62

### :white_check_mark: Checklist

- [x] Necessary tests added/updated?
- [ ] Necessary docs added/updated?
- [x] Code actually executed?
- [x] Vetting performed (unit tests, lint, etc.)?

### CODE REVIEW

File: components/ingest-service/integration_test/project_update_actions_test.go [hunk 1]
```diff
@@ -32,11 +39,11 @@ func TestProjectUpdateActionsPainlessElasticsearchScript(t *testing.T) {
 				Projects:         []string{"old_tag"},
 			},
 			projects: map[string]*iam_v2.ProjectRules{
-				"project9": &iam_v2.ProjectRules{
+				"project9": {
```
Comment (lancewf) [inline components/ingest-service/integration_test/project_update_actions_test.go]: These are all from `gofmt -s`

File: components/ingest-service/backend/elastic/elastic.go [hunk 2]
```diff
@@ -50,9 +50,14 @@ func (es *Backend) UpdateProjectTags(ctx context.Context, projectTaggingRules ma
 		return []string{}, errors.Wrap(err, "Failed to start Elasticsearch Node project tags update")
 	}
 
-	logrus.Debugf("Started Project rule updates with node job ID: %q", esNodeJobID)
+	esActionJobID, err := es.UpdateActionProjectTags(ctx, projectTaggingRules)
```
Comment (vjeffrey) [inline components/ingest-service/backend/elastic/elastic.go]: am i reading right that this was the only code change actually needed for this?

Comment (lancewf) [reply components/ingest-service/backend/elastic/elastic.go]: @vjeffrey That is correct! 😄

Comment (vjeffrey) [reply components/ingest-service/backend/elastic/elastic.go]: nice!!!

File: components/ingest-service/integration_test/project_update_actions_test.go [hunk 3]
```diff
@@ -337,3 +344,95 @@ func TestProjectUpdateActionsPainlessElasticsearchScript(t *testing.T) {
...
+			if event.Data.Fields["Completed"].GetBoolValue() {
+				complete = true
+			}
+		}
+	}
+
+	suite.RefreshIndices(suite.Indices()...)
+
+	// Test if the action was updated
+	actualActions, err := suite.GeActions(100)
```
Comment (vjeffrey) [inline components/ingest-service/integration_test/project_update_actions_test.go]: should this say GetActions? (not `GeActions`)

Comment (lancewf) [reply components/ingest-service/integration_test/project_update_actions_test.go]: 😆 Yes it should. I wonder how long that has been in there like that.

**Pull request merged**